interactions/lewd/lewd_datums.dm

```diff
@@ -245,6 +245,20 @@
 	require_target_bottomless = TRUE
 	max_distance = 0
 
+/datum/interaction/lewd/tribadism/display_interaction(mob/living/carbon/human/user, mob/living/carbon/human/target)
+	user.do_tribadism(target)
+
+/datum/interaction/lewd/tribadism
+	command = "tribadism"
+	description = "Grind your pussy against theirs."
+	interaction_sound = null
+	require_target_vagina = TRUE
+	require_user_vagina = TRUE
+	user_not_tired = TRUE
+	require_user_bottomless = TRUE
+	require_target_bottomless = TRUE
+	max_distance = 0
+
 /datum/interaction/lewd/mountass/display_interaction(mob/living/carbon/human/user, mob/living/carbon/human/target)
 	user.do_mountass(target)
 
```

Comment:
add the code here not in the middle of `mountass/display_interaction(...`

Reply:
Is this what you meant?

Reply:
yes, put the code you added bellow the `user.do_mountass(target)`

Reply:
Done.